tests/console/zypper_ar.pm

```diff
@@ -25,7 +25,7 @@ sub run {
         foreach (@repos_to_add) {
             next unless get_var("REPO_$_");    # Skip repo if not defined
             $repourl = $urlprefix . "/" . get_var("REPO_$_");
-            zypper_call "ar -c $repourl $_";
+            zypper_call "ar -c $repourl $_" if zypper_call("lr | grep $_", allow_exit_codes => [1]);
```

Comment:
beware: 'OSS' matches on the repo NON_OSS (so if NON_OSS would be added, but OSS not, you'd still skip adding it here)

you probably want rather something like ```grep '\s$_\s'```